dev/io.openliberty.accesslists.internal/src/io/openliberty/accesslists/AccessListKeysFacade.java

```diff
@@ -10,33 +10,44 @@
  *******************************************************************************/
 package io.openliberty.accesslists;
 
+import java.util.Arrays;
+
 /**
- * This interface enables clients of the AccessList code to succinctly
- * provide a means to obtain a set of access lists from configuration
- * objects without this bundle having to be aware of the concrete types
- * in the client code. The default implmentations allow for there to be
- * 'missing' keys in particular clients (e.g. no hosts in UDP).
+ * This interface enables clients of the AccessList code to succinctly provide a
+ * means to obtain a set of access lists from configuration objects without this
+ * bundle having to be aware of the concrete types in the client code. The
+ * default implmentations allow for there to be 'missing' keys in particular
+ * clients (e.g. no hosts in UDP).
  */
 public interface AccessListKeysFacade {
 
-	default String[] getAddressExcludeList() {
-		return null;
-	}
+    default String[] getAddressExcludeList() {
+        return null;
+    }
+
+    default String[] getHostNameExcludeList() {
+        return null;
+    }
+
+    default String[] getAddressIncludeList() {
+        return null;
+    };
 
-	default String[] getHostNameExcludeList() {
-		return null;
-	}
+    default String[] getHostNameIncludeList() {
+        return null;
+    }
 
-	default String[] getAddressIncludeList() {
-		return null;
-	};
+    default boolean getCaseInsensitiveHostnames() {
+        return true;
+    }
 
-	default String[] getHostNameIncludeList() {
-		return null;
-	}
+    default String asString() {
+        return  "\n getAddressExcludeList()"  + Arrays.toString(getAddressExcludeList()) +"\n" +
+                " getHostNameExcludeList()"+ Arrays.toString(getHostNameExcludeList()) +"\n" +
```

Comment:
inconsistent spacing for some of this formatting - we probably want a space before each `Arrays.toString()`

Reply:
Will do, (I learned that an interface default method cannot override a method from j.l.Object like toString.)